.github/workflows/release.yaml

```diff
@@ -210,6 +210,11 @@ jobs:
           git commit -m "[Release] Update Chart.yaml"
           git push
 
+      - name: Summary
+        run: |
+          echo "**Upgrade with helm:**" >> $GITHUB_STEP_SUMMARY
```

Comment:
I think you first need to run `helm repo update castai-helm` to update local repo, or does helm upgrade fetches latest chart?

Reply:
Good point, I totally forgot about it. Thanks!